src/vcpu.rs

```diff
@@ -506,3 +306,29 @@ impl<H: AxVCpuHal> RISCVVCpu<H> {
         }
     }
 }
+
+#[inline(always)]
+fn sbi_call_legacy_0(eid: usize) -> usize {
```

Comment:
have you verified that this `sbi_call` works in the latest version of qemu?

Reply:
I just trtied 9.2.0. It works fine.